test/core/compression/BUILD

```diff
@@ -42,6 +43,26 @@ grpc_cc_test(
     uses_polling = False,
 )
 
+grpc_fuzzer(
+    name = "message_compress_fuzzer",
+    srcs = ["message_compress_fuzzer.cc"],
+    corpus = "message_compress_corpus",
+    deps = [
```

Comment:
The fuzzer tests are currently failing on windows for compilation reasons.

Please add 
`tags = ["no_windows"],`
to each of the fuzzer build targets to resolve this